fetch_and_store_cron.py

```diff
@@ -41,28 +41,22 @@
                 cleaned_url = site_url.replace("https://", "")
                 projects.add(cleaned_url)
 
-# --- Date range for last 3 complete years ---
-today = datetime.utcnow().date()
-# Get the first day of current month
-current_month_start = today.replace(day=1)
-# Get the last day of previous month (end of data range)
-end_date = current_month_start - timedelta(days=1)
+# --- Date range for last month ---
+today = datetime.utcnow().date().replace(day=1)
+last_month_end = today - timedelta(days=1)
+last_month_start = last_month_end.replace(day=1)
 
-# Go back 3 years from the end date to get start date
-start_year = end_date.year - 3
-start_date = datetime(start_year, 1, 1).date()  # January 1st, 3 years ago
-
-start = start_date.strftime("%Y%m%d")
-end = end_date.strftime("%Y%m%d")
+start = last_month_start.strftime("%Y%m%d")
+end = last_month_end.strftime("%Y%m%d")
 
 # --- Connect to Toolforge DB ---
-DB_NAME = 'community_alerts'
-DB_TABLE = 'edit_counts'
+DB_NAME = "s56391__community_alerts"
```

Comment:
This should be come from config file. So based on env file, it can switch for dev and prod.